byron/ledger/executable-spec/test/Ledger/Delegation/Properties.hs

```diff
@@ -267,7 +267,7 @@ dcertsAreTriggered :: Property
 dcertsAreTriggered = withTests 300 $ property $
   -- The number of tests was determined ad-hoc, since the default failed to
   -- uncover the presence of errors.
-  forAll nonTrivialTrace >>= dcertsAreTriggeredInTrace
```

Comment:
Is there an issue for this PR in particular? I'm failing to see how a specific property relates to rewriting the way traces are generated.